_delphi_utils_python/data_proc/geomap/geo_data_proc.py

```diff
@@ -391,6 +391,10 @@ def create_fips_population_table():
     df_pr = df_pr.groupby("fips").sum().reset_index()
     df_pr = df_pr[~df_pr["fips"].isin(census_pop["fips"])]
     census_pop_pr = pd.concat([census_pop, df_pr])
+
+    # Zero out the populations for the state FIPS codes XX000 to avoid double counting
+    megafips_codes = [str(x).zfill(2) + "000" for x in range(1, 73)]
```

Comment:
Won't this result in inf results for megafips prop signals?

Reply:
I don't think so. The denominator in JHU is obtained my merging a FIPS population dataframe on the FIPS-level data, which is then aggregated to the state level. I verified this in [this updated notebook](https://github.com/cmu-delphi/covidcast-indicators/blob/296d75da2488d70672a7db30418ccad313c1c045/testing_utils/indicator_validation.template.ipynb), where I compare the new JHU state prop values with USA facts. Rhode Island is the largest outlier because we are now using values from Unassigned that we weren't before.

Let me double check how other indicators like Safegraph do this calculation, though.

Reply:
I meant county files -- don't we include unassigned as a megafips for county?

Reply:
> don't we include unassigned as a megafips for county?

Turns out we don't! So this won't be a problem.

Reply:
We do, in fact:
```
$ zgrep -m1 "^[0-9][0-9]000" *_county_* | head
20200220_county_confirmed_7dav_cumulative_num.csv.gz:60000,0.0,NA,NA
20200220_county_confirmed_7dav_incidence_num.csv.gz:60000,0.0,NA,NA
20200220_county_confirmed_7dav_incidence_prop.csv.gz:01000,0.0,NA,NA
20200220_county_confirmed_7day_avg_cumulative_num.csv.gz:72000,0.0,NA,NA
20200220_county_confirmed_7day_avg_cumulative_prop.csv.gz:72000,0.0,NA,NA
20200220_county_confirmed_7day_avg_incidence_num.csv.gz:72000,0.0,NA,NA
20200220_county_confirmed_7day_avg_incidence_prop.csv.gz:72000,0.0,NA,NA
20200220_county_confirmed_cumulative_num.csv.gz:60000,0.0,NA,NA
20200220_county_confirmed_incidence_num.csv.gz:60000,0.0,NA,NA
20200220_county_confirmed_incidence_prop.csv.gz:01000,0.0,NA,NA
```